x-pack/plugins/ml/public/application/data_frame_analytics/pages/analytics_management/components/models_management/expanded_row.tsx

```diff
@@ -0,0 +1,351 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import React, { FC, Fragment } from 'react';
+import { FormattedMessage } from '@kbn/i18n/react';
+import {
+  EuiDescriptionList,
+  EuiPanel,
+  EuiSpacer,
+  EuiTabbedContent,
+  EuiTitle,
+  EuiNotificationBadge,
+  EuiFlexGrid,
+  EuiFlexItem,
+  EuiCodeBlock,
+  EuiText,
+  EuiFlexGroup,
+  EuiStat,
+} from '@elastic/eui';
+import { ModelItemFull } from './models_list';
+
+interface ExpandedRowProps {
+  item: ModelItemFull;
+}
+
+export const ExpandedRow: FC<ExpandedRowProps> = ({ item }) => {
+  const {
+    inference_config: inferenceConfig,
+    stats,
+    metadata,
+    tags,
+    version,
+    // eslint-disable-next-line @typescript-eslint/naming-convention
+    estimated_operations,
+    // eslint-disable-next-line @typescript-eslint/naming-convention
+    estimated_heap_memory_usage_bytes,
+    // eslint-disable-next-line @typescript-eslint/naming-convention
+    default_field_map,
+    // eslint-disable-next-line @typescript-eslint/naming-convention
+    license_level,
+    pipelines,
+  } = item;
+
+  const details = {
+    tags,
+    version,
+    estimated_operations,
+    estimated_heap_memory_usage_bytes,
+    default_field_map,
+    license_level,
+  };
+
+  function formatToListItems(items: Record<string, any>) {
+    return Object.entries(items)
+      .map(([title, value]) => ({
+        title,
+        description: typeof value === 'object' ? JSON.stringify(value) : value,
+      }))
+      .filter(({ description }) => {
+        return description !== undefined;
+      });
+  }
+
+  const tabs = [
```

Comment:
As noted by @qn895, a different font size is used for the tabs in the expanded rows of the Jobs and Models tabs. I prefer the smaller size used on the Jobs tab, although I see the larger size (16px) is used for the Anomaly detection jobs list.

Reply:
Changed the size of the tabs in https://github.com/elastic/kibana/pull/74978/commits/0b63d74600db3290a507569e1c9f71b4583bdae7